airdrop-claim-reject-cancel-hts-token-js-sdk/utils/queries.js

```diff
@@ -0,0 +1,28 @@
+import { TransactionRecordQuery, TokenInfoQuery, AccountBalanceQuery } from "@hashgraph/sdk";
+
+export async function txRecQueryFcn(txId, client) {
+	const recQuery = await new TransactionRecordQuery().setTransactionId(txId).setIncludeChildren(true).execute(client);
+	return recQuery;
+}
+
+export async function tokenQueryFcn(tkId, client) {
+	let info = await new TokenInfoQuery().setTokenId(tkId).execute(client);
+	return info;
+}
+
+export async function balanceCheckerFcn(acId, tkId, client) {
+	let balanceCheckTx = [];
+	try {
+		balanceCheckTx = await new AccountBalanceQuery().setAccountId(acId).execute(client);
+		console.log(
+			`- Balance of account ${acId}: ${balanceCheckTx.hbars.toString()} + ${balanceCheckTx.tokens._map.get(tkId.toString())} unit(s) of token ${tkId}`
```

Comment:
❓ about the `_map` function here - is it an officially supported API of the SDK?

If not, we should avoid using in, and prefer to instead write our own loop/ lookup logic.

Reply:
Supported. `AccountBalance.tokens` field is likely to be deprecated in the future, but not an immediate concern.